Src/Sarif.PatternMatcher.Function/HttpAnalyzeFn.cs

```diff
@@ -0,0 +1,81 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+using System;
+using System.IO;
+using System.Threading.Tasks;
+
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Extensions.Http;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.CodeAnalysis.Sarif.PatternMatcher.Function
+{
+    public static class HttpAnalyzeFn
```

Comment:
>HttpAnalyzeFn [](start = 24, length = 13)

Can we spell this out? Generally, acronyms and abbreviations aren't used in public .NET constructs. 'HttpAnalyzeFunction'. Change the file name too?